### ISSUE TITLE
[Beta] Android N support

### ISSUE THREAD

**Issue by speed2O** [StephenBlackWasAlreadyTaken/xDrip-Experimental#337]
I dove headfirst into the Dev edition. Everything looks fine on the outside but I'm not receiving info in xdrip from my Dexcom with share. It shows connected in app but no data has shown up. I'll keep looking for a solution. Hopefully someone eles can help.

**Comment by gayprogrammer** [comment]
Working for me on Android N build NPD35K Nexus 6P.with xdrip beta 2.0.5_2-7a93a06.

Make sure that Location permission is enabled in the system settings app.

**Comment by speed2O** [comment]
Great to hear!!! I'm running Nexus 6(shamu)  and I did have the location permission enabled, but xdrip beta2.0.5_2-7a93a06 did not work on my Nexus6 with NPC91K, NPC91O (preview 2). I rolled back to stock Android 6.0.1. I am downloading NPS35K (preview 3) right now hopefully things will work better this time. I will post an update. Did you use preview 2, did it work for you? Thanks for the inspiration.
